rules/windows/process_creation/proc_creation_win_indirect_command_execution_forfiles.yml

```
@@ -0,0 +1,32 @@
+title: Indirect Command Exectuion via Forfiles
+id: a85cf4e3-56ee-4e79-adeb-789f8fb209a8
+status: experimental
+description: Detects the use of native Windows tool, forfiles to execute a file. Adversaries may abuse utilities that allow for command execution to bypass security restrictions that limit the use of command-line interpreters.
+author: Tim Rauch (rule), Elastic (idea)
+references:
+    - https://github.com/elastic/protections-artifacts/commit/746086721fd385d9f5c6647cada1788db4aea95f#diff-73d61931b2c77fde294189ce5d62323b416296a7c23ea98a608f425566538d1a
+    - https://lolbas-project.github.io/lolbas/Binaries/Forfiles/
+date: 2022/10/17
+tags:
+    - attack.defense_evasion
+    - attack.t1202
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        ParentImage|endswith: '\forfiles.exe'
+        ParentCommandLine|contains|all:
+            - ' /c '
+            - ' /p '
+            - ' /m '
+    filter:
+        Image|endswith: '\cmd.exe'
+        CommandLine|contains|all:
+            - 'xcopy'
+            - 'cmd /c del @PATH'
+            - 'move'
+    condition: selection and not filter
```

```suggestion
    selection_parent:
        ParentImage|endswith: '\forfiles.exe'
    selection_c:
        ParentCommandLine|contains:
            - ' /c '
            - ' -c '
    selection_p:
        ParentCommandLine|contains:
            - ' /p '
            - ' -p '
    selection_m:
        ParentCommandLine|contains:
            - ' /m '
            - ' -m '
    filter:
        Image|endswith: '\cmd.exe'
        CommandLine|contains|all:
            - 'xcopy'
            - 'cmd /c del 
```

Since the `-` can also be used.